napalm_nxos/nxos.py

```diff
@@ -57,6 +69,7 @@ def __init__(self, hostname, username, password, timeout=60, optional_args=None)
         self.changed = False
         self.replace_file = None
         self.merge_candidate = ''
+        self.ssh_connection = False
 
```

Comment:
`self.timeout` seems to be ignored, shouldn't we use it to set the initial timer? I also wonder if we could use that to replicate the `global_delay_factor` effect to avoid having an extra argument. @ktbyers?